templates/confidential-computing/index.html

```diff
@@ -0,0 +1,287 @@
+{% extends "templates/base.html" %}
+
+{% block title %}Confidential Computing{% endblock %}
+{% block meta_description %}Protect data in use with confidential computing. Build the foundation of your
+privacy-enhancing technology strategy with Ubuntu confidential VMs on both public and private clouds.{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1QGwS5CYlwXkaPibSFvcag6zLcUH5lNDUbM18hCqbOJM/edit{% endblock %}
+
+{% block outer_content %}
+<div class="is-paper">
+  <section class="p-strip is-shallow u-no-padding--bottom">
+    <div class="row">
+      <div class="col-9 col-start-large-4">
+        <div class="p-section">
```

Comment:
```suggestion
        <div class="p-section--shallow">
```

Reply:
if I do so, the bottom padding becomes 24px instead of 64px as in Figma

Reply:
yes that's how much is needed there. This is a wireframe, hasn't gone through visual as I'm on the analyst day slides and we agreed any corrections will happen in the review process. Apologies for that - the alternative would have beento pause that until after next week, which I understand is a bigger problem - as the event is today. So some things are a little off in the wireframe.